src/Administration/Resources/app/administration/vite.config.mts

```diff
@@ -73,6 +73,7 @@ export default defineConfig(({ command }) => {
                     secure: false,
                 },
             },
+            origin: `${process.env.APP_URL.replace(/:\d+$/, "")}:` + (Number(process.env.ADMIN_PORT) || 5173),
```

Comment:
Can we add this as an opt-in to make sure that no other setup will break (local, docker, etc.)?

Reply:
See my changes - I now check for DDEV_PRIMARY_URL env variable, which should only be set in ddev environment, and use this as server.origin.

I assumed APP_URL would do for all environments, but testing in native docker setup I realized it would not.

Must say that I tested this only for ddev and docker setups myself, so remains to be verified for dockware and devenv.